src/BlockTypes/ProductQuery.php

```diff
@@ -710,5 +712,42 @@ private function get_global_query( $parsed_block ) {
 		return $query;
 	}
 
-}
+	/**
+	 * Return a query that filters products by rating.
+	 *
+	 * @return array
+	 */
+	private function get_filter_by_rating_query() {
+		$filter_rating_values     = get_query_var( RatingFilter::RATING_QUERY_VAR );
+		$product_visibility_terms = wc_get_product_visibility_term_ids();
+
+		if ( empty( $filter_rating_values ) ) {
+			return array();
+		}
+
+		$parsed_filter_rating_values = explode( ',', $filter_rating_values );
+
+		if ( empty( $parsed_filter_rating_values ) || empty( $product_visibility_terms ) ) {
+			return array();
+		}
 
+		$rating_query = array_map(
```

Comment:
I think `$rating_terms` better explains the value of this variable.
```suggestion
		$rating_terms = array_map(
```